src/Microsoft.DocAsCode.Build.Engine/DocumentBuilderCore.cs

```diff
@@ -0,0 +1,781 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Composition.Hosting;
+    using System.Collections.Immutable;
+    using System.Reflection;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Exceptions;
+    using Microsoft.DocAsCode.Plugins;
+    using Microsoft.DocAsCode.Utility;
+
+    public class DocumentBuilderCore : IDisposable
```

Comment:
Can't be internal class since it's used by MetadataMerger

---
In reply to: [71101441](https://github.com/dotnet/docfx/pull/501#discussion_r71101441) [](ancestors = 71101441)